src/style.css

```
@@ -1 +1,61 @@
 /* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
```

Only set box-sizing in * selector, move margin and padding to body selector instead

```suggestion
  box-sizing: border-box;
```